wiki/Tournaments/SS_Tournament/2018/en.md

```diff
@@ -0,0 +1,205 @@
+* Staff is not allowed to play excluding commentators and streamers.
+* Group Stage Matches will be best of 5
+* Round of 16 will be best of 7
+* Quarter Finals and Semi Finals will be best of 9
+* Finals will be best of 11
+* Grand Finals will be best of 13
+* In the case of a tie in group stage, team advancement will be based off a team's '[map wins - map losses = Points]' where higher points advance. If the score is equal, the score of Team A Vs Team B will be taken into account, i.e. Team A and Team B both have 6 wins and 6 losses, but Team A beat Team B 3-2 Team A will advance.If there is still a tie it will be average score difference gathered by the formula '[Team A's scores of all played maps - All opposing teams scores / maps played]'.If there is still a tie after this, it will be based off seeding.
+
+### Match Rules
+
+1. The designated referee will have the lobby open using the format SST: (blue team) vs. (red team)
+1. The referee will message the team captains or anyone online from the team.
+1. Teams will join here. Once all 6 players are in the referee will ask for their warm ups. Order in which they are played is irrelevant. (this step is skipped for group stages)
+1. Both teams will '!roll', the higher number rolled will be the team who will ban first 
+1. Both teams will alternate banning maps until both teams have banned out 2 maps
+1. The team that lost the roll is the team that picks first and it it alternates from there. 
+Example: BLUE rolls 67 RED rolls 10 the order would be 
+Blue BAN Red BAN  Blue BAN Red BAN : Red MAP Blue MAP Red MAP Blue MAP …
+1. Teams have 60 seconds to pick a map and 120 seconds to ready up refs will use '!mp timer' to keep time if a team doesn't pick in the first 60 seconds the opposing team will pick instead if the players are not ready after 120 seconds the ref will at their discretion start the match *Refs are in full control of any leniency here and may choose not to enforce this until semifinals at which point enforcement will be mandatory* 
+1. Once a map has been picked and everyone is ready the map will start after 5 seconds using '!mp start 5'
+1. If a player disconnects within the first 15 seconds of a map the referee will abort the match and the map will be restarted. Anyone found abusing this will be disqualified from the tournament. This may only happen once per team per match after the first dc the referee is not to abort
+
+
+## Mappools
+
+### Qualifiers
+
+![Qualifiers Mappool](Qaulifiers.jpg)
+
+- NoMod
+  - [Ceui - COLORFUL DAYS!!](https://osu.ppy.sh/b/203315) [Insane]
+  - [Mastermind(xi+nora2r) - Dreadnought](https://osu.ppy.sh/b/813331) [Another]
+  - [Function Phantom - Euclid] (https://osu.ppy.sh/b/971086) [toybot's Insane]	
```

Comment:
omit space in-between `]` and `(`

Reply:
`[Function Phantom - Euclid] (https://osu.ppy.sh/b/971086) [toybot's Insane]` ->
`[Function Phantom - Euclid](https://osu.ppy.sh/b/971086) [toybot's Insane]`